components/log_wrappers/src/lib.rs

```diff
@@ -0,0 +1,148 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Provides wrappers for types that comes from 3rd-party and does not implement slog::Value.
+
+extern crate hex;
+extern crate kvproto as lib_kvproto;
+#[macro_use]
+extern crate slog;
+extern crate slog_term;
+
+pub mod test_util;
+
+// We don't introduce namespace std for std functions.
+
+pub mod time {
```

Comment:
As a result, you can also remove this mod.

Reply:
Reserved for supporting `Option<Duration>`

Reply:
Updated, make it generic so that it can be used with any `T: Display` or `T: Debug`.